testsupport/tiers/checks.go

```diff
@@ -271,6 +275,57 @@ func (a *testTierChecks) GetClusterObjectChecks() []clusterObjectsCheck {
 	return []clusterObjectsCheck{}
 }
 
+type appstudioTierChecks struct {
+	baseTierChecks
+}
```

Comment:
The `appstudio` tier is not based on the `base` tier, so we should not reuse it
```suggestion
type appstudioTierChecks struct {
	tierName string
}
```